etc/kayobe/inventory/group_vars/controllers/firewall

```diff
@@ -0,0 +1,375 @@
+---
+
+firewallgen_enable_firewall: True
+
+firewallgen_ipv4_input_allow_rules:
+  - interface: "all"
+    port: 111
+    proto: tcp
+    comment: "hint: used by 'systemd'"
+  - interface: "lo"
+    port: 25
+    proto: tcp
+    destination: "127.0.0.1"
+    comment: "hint: used by 'master'"
+  - interface: "lo"
+    port: 6633
+    proto: tcp
+    destination: "127.0.0.1"
+    comment: "hint: used by 'neutron-openvsw' in docker container 'neutron_openvswitch_agent'"
+  - interface: "lo"
+    port: 6640
+    proto: tcp
+    destination: "127.0.0.1"
+    comment: "hint: used by 'ovsdb-server' in docker container 'openvswitch_db'"
+  - interface: "{{ admin_oc_net_interface }}"
+    port: 22
+    proto: tcp
+    comment: "hint: used by 'sshd'"
```

Comment:
We should probably extract the "common" stuff out somehow into a different list, then join the lists together? We have the ceph servers and a few other groups to do here too.

Reply:
yeah, I wasn't thinking of it like that. its generated, so screw it.